apps/tailwind-components/components/Button.vue

```diff
@@ -44,6 +44,12 @@ const SIZE_MAPPING = {
   medium: "h-14 px-7.5 text-heading-xl gap-4",
   large: "h-18 px-8.75 text-heading-xl gap-5",
 };
+const ICON_SIZE_MAPPING = {
+  tiny: 12,
```

Comment:
there is no way tailwind takes care of this?

Reply:
Leave it for now and fix in another PR